XLS-0089-multi-purpose-token-metadata-schema/README.md

```diff
@@ -1,18 +1,18 @@
 <pre>
-Title:       <b>Multi-Purpose Token Metadata Schema</b>
-Revision:    <b>Draft</b> (2025-06-27)
-
-Authors:     <a href="mailto:shawnxie@ripple.com">Shawn Xie</a>
-             <a href="mailto:gtsipenyuk@ripple.com">Greg Tsipenyuk</a>
-             <a href="mailto:smittal@ripple.com">Shashwat Mittal</a>
-             <a href="mailto:jberridi@ripple.com">Julian Berridi</a>
-
-Affiliation: <a href="https://ripple.com">Ripple</a>
+  xls: 89
+  title: Multi-Purpose Token Metadata Schema
+  description: Standardized metadata schema for Multi-Purpose Tokens to improve discoverability, comparability, and interoperability
+  author: Nathan Nichols <nnichols@ripple.com>, Shawn Xie <shawnxie@ripple.com>
```

Comment:
Shawn Xie, Greg Tsipenyuk, Shashwat Mittal, Julian Berridi are all in the original version of this file